src/components/progress-indicator/index.js

```diff
@@ -0,0 +1,231 @@
+import React, { useMemo } from 'react'
+import PropTypes from 'prop-types'
+import clsx from 'clsx'
+
+import { makeStyles } from '../../utils/make-styles'
+import { CheckBold } from '../../icons'
+import Spin from './spin'
+
+
+const customClasses = (size) => {
+  return makeStyles({
+    progressIndicatorRoot: {
+      '& .svg-icon': {
+        width: `${size/1.875}rem`,
+      },
+      '& .text': {
+        fontFamily: 'PT Sans',
+      },
+      '& .caption': {
+        fontFamily: 'PT Sans',
+        fontSize: `${size/2.14}rem`,
+      },
+      '& .indicator-h': {
+        width: `${(size/0.20) + 2}rem`,
+      },
+      '& .indicator-v': {
+        height: `${size/0.25}rem`,
+      },
+      '& .label-h': {
+        marginTop: '0.313rem',
+        fontSize: `${size/1.875}rem`,
+        lineHeight: `${size/1.875}rem`,
+        letterSpacing: '0.05em',
+      },
+      '& .label-v': {
+        marginLeft: '0.313rem',
+        fontSize: `${size/1.875}rem`,
+        letterSpacing: '0.05em',
+      },
+      '& .number-complete': {
+        width: `${size}rem`,
+        height: `${size}rem`,
+        padding: '0.313rem',
+        borderRadius: '50%',
```

Comment:
Applying `border-radius` without resetting the border prop would produce a bit of a jagged surface (for filled/complete circles)
![before 1](https://user-images.githubusercontent.com/2837532/204615846-c0d9f993-1297-4151-8eb7-faeaf0adb121.png)

![before 2](https://user-images.githubusercontent.com/2837532/204615600-f157e822-5df6-48a1-bd00-5a61a668e84f.png)


Add `border: 0;` for a smoother edge of a circle:
![after 1](https://user-images.githubusercontent.com/2837532/204616110-00e877a0-0d61-434e-88b1-9efebcec6226.png)

![after 2](https://user-images.githubusercontent.com/2837532/204615549-847456d3-a3b5-4650-903c-548304a4f3de.png)